examples/suite-classification/project.clj

```diff
@@ -5,7 +5,33 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [thinktopic/cortex.suite "0.3.1-SNAPSHOT"]
-                 ;;If you have cuda-8.0 installed then add this:
-                 [thinktopic/gpu-compute "0.3.1-SNAPSHOT"]]
+                 ;;Default way of displaying anything is a web page.
+                 ;;Because if you want to train on aws (which you should)
+                 ;;you need to get simple servers up and running easily.
+                 [thinktopic/think.gate "0.1.1-SNAPSHOT"]
```

Comment:
Is there a way to not depend on the think.gate snapshot since it is not a bundled project?